misc/install.func

```diff
@@ -166,21 +192,57 @@ fi
 EOF
   chmod +x /usr/local/bin/apt-proxy-detect.sh
   fi
-  $STD apt-get update
-  $STD apt-get -o Dpkg::Options::="--force-confold" -y dist-upgrade
-  rm -rf /usr/lib/python3.*/EXTERNALLY-MANAGED
+  if grep -qEi "debian|ubuntu" /etc/os-release; then
+     $STD apt-get update
+     $STD apt-get -o Dpkg::Options::="--force-confold" -y dist-upgrade
+   # RPM-based systems
+   elif grep -qEi "fedora|centos|rocky|almalinux" /etc/os-release; then
+     $STD dnf -y upgrade
+   # openSUSE
+   elif grep -qEi "opensuse" /etc/os-release; then
+     $STD zypper -n update
+   else
+     msg_error "Unsupported OS detected!"
+     exit 1
+   fi
   msg_ok "Updated Container OS"
 }
 
 # This function modifies the message of the day (motd) and SSH settings
 motd_ssh() {
-  echo "export TERM='xterm-256color'" >>/root/.bashrc
-  echo -e "$APPLICATION LXC provided by https://helper-scripts.com/\n" >/etc/motd
-  chmod -x /etc/update-motd.d/*
-  if [[ "${SSH_ROOT}" == "yes" ]]; then
-    sed -i "s/#PermitRootLogin prohibit-password/PermitRootLogin yes/g" /etc/ssh/sshd_config
-    systemctl restart sshd
+  # Set terminal to 256-color mode if not already set
+  grep -qxF "export TERM='xterm-256color'" /root/.bashrc || echo "export TERM='xterm-256color'" >> /root/.bashrc
```

Comment:
What was the conclusion here?

Reply:
I had sporadic problems with it not running. In other words, this is just a backup in case it didn't work. 

The problem only occurs under Ubuntu 24.04 / 24.10.

Reply:
So it works on Ubuntu 24.04 / 24.10 with this check, e.g. `export TERM='xterm-256color'` is already set in `.bashrc` in those distros?

Reply:
yep